corpus/accounts/models.py

```diff
@@ -151,3 +155,22 @@ def save(self, *args, **kwargs):
 
     def __str__(self):
         return f"{self.user.first_name} {self.user.last_name} [{self.sig.name}]"
+
+
+class Core(models.Model):
+    user = models.OneToOneField(User, null=False, on_delete=models.CASCADE)
+    post = models.CharField(max_length=100, null=False)
+    sig = models.ForeignKey(SIG, null=False, on_delete=models.CASCADE)
+    term_start = models.DateField(default=now, null=False)
```

Comment:
Let's not specify a default value here. It's very rarely the case that someone is sitting *during* core transfer and creating/updating a core post object on Corpus xD.
Additionally, `null=False` is the Django default value, so its not necessary here.